vmdb/app/models/ems_microsoft.rb

```diff
@@ -96,7 +96,8 @@ def execute_power_operation(cmdlet, vm_uid_ems, *parameters)
     return unless vm_uid_ems.guid?
 
     params  = parameters.join(" ")
-    command = "powershell #{cmdlet}-SCVirtualMachine -VM (Get-SCVirtualMachine -ID #{vm_uid_ems}) #{params}"
+    command = "powershell Import-Module VirtualMachineManager; Get-VMMServer #{ipaddress}; "
```

Comment:
* Does `Get-VMMServer` support hostnames and ip addresses?

* If so, we should use `Get-VMMServer #{hostname}` and let users specify the ipaddress in the hostname field if they need to use ipv4/ipv6 (see #1800)

* If so, does it support IPv6 literal addresses such as: `Get-VMMServer ::1`;  For example, IPv6 addresses need to be left bare for Socket connections, but wrapped in `[]` in URIs:
  * `TCPSocket.open("::1", 80)`
  * `wget http://[::1]:80/blah`